sample/src/App.tsx

```diff
@@ -36,11 +36,11 @@ Sentry.init({
   onReady: ({didCallNativeInit}) => {
     console.log('onReady called with didCallNativeInit:', didCallNativeInit);
   },
-  maxBreadcrumbs: 150, // Extend from the default 100 breadcrumbs.
```

Comment:
JS does not allow more than 100 crumbs, there's an absolut number.
So either we remove it or just set it to 100.